.github/workflows/enos-test-ui.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: Vault UI Tests
```

Comment:
Similar to how we're renaming in the replication PR, would you mind updating the workflow and name to lead with `test`, eg: `test-enos-scenario-ui`?

Reply:
The idea being here that not everybody knows what enos is, but they probably have a good idea what `test` means.

And since we're adding a lot of these it would be nice to have a common prefix.